src/core_modules/capture-core/components/WidgetsChangelog/common/ChangelogFilterBar/ChangelogFilterBar.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import React, { useMemo, useState, useCallback } from 'react';
+import i18n from '@dhis2/d2-i18n';
+import { withStyles } from '@material-ui/core/styles';
+import { spacers } from '@dhis2/ui';
+import type { ChangelogFilterProps, FilterValueType } from './ChangelogFilter.types';
+import { DropdownFilter } from './DropdownFilter';
+import { FIELD_TYPES, FILTER_TARGETS, CHANGELOG_ENTITY_TYPES } from '../Changelog/Changelog.constants';
+
+const styles = {
+    container: {
+        padding: `0 0 ${spacers.dp8} 0`,
+    },
+};
+
+const getFilterField = (id: string, entityType: string) => {
+    if (entityType === CHANGELOG_ENTITY_TYPES.TRACKED_ENTITY) {
+        return FILTER_TARGETS.ATTRIBUTE;
+    }
```

Comment:
In the Profile widget, the change log for a TET coordinate (both point and polygon) is missing. Looking at the epic, I think this was never implemented in the API. Do we want to add it?
The TET geometry appears in the filter options dropdown, but throws a 400 error when selected. The proper API request should probably look like this: `/tracker/trackedEntities/[trackedEntityId]/changeLogs?filter=field:eq:geometry`, but I don't think this type of filter is implemented in the API either. Any insights on this? 🤔 

<img width="2474" alt="Screenshot 2025-01-20 at 16 15 27" src="https://github.com/user-attachments/assets/ebf6c984-992e-4023-8eb4-06a56fc33fb8" />

Thanks!

Reply:
TET-geometry will not be implemented at this point. If we want it, we will have to create a separate issue for it. So, we should remove TET geometry from the filter drop down.